website/config.toml-E

```diff
@@ -0,0 +1,241 @@
+baseURL = "/"
```

Comment:
Was this file committed deliberately?

Reply:
I added everything which got generated. Do we need to remove this file?

Reply:
If this was a one off issue, let's just delete this file, but if there's a bug in the generation we should fix that. Once we know the cause, if it's trivial let's fix here, otherwise we can file an issue on the 1.21 release milestone and ignore for today.

Reply:
Raised issue for the same. https://github.com/operator-framework/operator-sdk/issues/5713